website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -55,12 +55,12 @@
                             Now that you've shared your ${document_type}, take advantage of more OSF features:
                             <ul>
                                 % if has_psyarxiv_chronos_text:
-                                    <li>Submit your preprint to an APA-published journal. <a href="https://help.osf.io/article/401-submitting-to-a-moderated-preprint-service ">Learn more</a></li>
+                                    <li>Submit your preprint to an APA-published journal. Learn about <a href="https://help.osf.io/article/376-preprints-home-page#Submitting-to-a-Moderated-Preprint-Service-nueKG">Submitting to a Moderated Preprint Service</a>.</li>
```

Comment:
Note that I haven't actively tested each and every link in this site (this whole project is drive-by, I care, but I don't expect to be a real user). But, as we're trying to  fix some `Read more` links below, and switching some to `Learn how...` it became clear that these other links should get some love. I tried to use this link and it redirected -- I've included the anchor to make the link vaguely useful.